pitop/robotics/drive_controller.py

```diff
@@ -1,56 +1,66 @@
-from math import floor, pi
+from math import (
+    floor,
+    pi,
+    radians,
+)
+from time import sleep
+
+from simple_pid import PID
 
-from pitop import (
+from pitop.core.exceptions import UninitializedComponent
+from pitop.core.mixins import (
+    Stateful,
+    Recreatable,
+)
+from pitop.pma import (
     EncoderMotor,
     ForwardDirection,
 )
 
-from pitop.system.port_manager import PortManager
-from simple_pid import PID
 
+class DriveController(Stateful, Recreatable):
+    """Abstraction of a vehicle with two wheels connected by an axis, and an
```

Comment:
Let's avoid "abstraction of", and just describe straight away. It doesn't add any value. "Represents" would be better.